addons/binding/org.openhab.binding.onkyo/README.md

```diff
@@ -0,0 +1,78 @@
+# Onkyo Binding
+
+This binding integrates the Onkyo AV receivers.
+
+## Introduction
+Binding should be compatible with Onkyo AV receivers which support ISCP (Integra Serial Control Protocol) over Ethernet (eISCP).
+
+## Supported Things
+This binding supports only one thing: The Onkyo AV Receiver
+
+
+## Discovery
+This binding can discover the supported Onkyo AV Receivers. At the moment only the following models are supported:
+
+* TX-NR535
```

Comment:
Is there some sort of constraint such that only the TX-NR535 is supported? Or is that the only one this has been tested with?